File: SkaiLab.Invoice/ClientApp/src/app/models/organisation-user.ts
import { MenuFeature } from "./menu-feature";

export class OrganisationUser{
    organisationId:string;
    roleName:string;
    isInviting:boolean;
    isInvitingExpired:boolean;
    menuFeatures:MenuFeature[];
    isOwner:boolean;
    user:User;
    isAdministrator:boolean;
    constructor(){
        this.organisationId="";
        this.roleName="";
        this.isInviting=true;
        this.isInvitingExpired=false;
        this.menuFeatures=[];
        this.isOwner=false;
        this.user=new User();
        this.isAdministrator=true;
    }
}
export class User{

    firstName:string;
    lastName:string;
    id:string;
    email:string;
    name:string;
    phoneNumber:string;
    constructor(){
        this.firstName="";
        this.lastName="";
        this.id="";
        this.email="";
        this.name="";
        this.phoneNumber="";
    }
}

File: SkaiLab.Invoice.Models/OrganisationUser.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SkaiLab.Invoice.Models
{
    public class OrganisationUser
    {
        public string OrganisationId { get; set; }
        public User User { get; set; }
        public string RoleName { get; set; }
        public bool IsInviting { get; set; }
        public bool IsInvitingExpired { get; set; }
        public List<MenuFeature> MenuFeatures { get; set; }
        public bool IsOwner { get; set; }
        public bool IsAdministrator { get; set; }
    }
}
